main/lsp/lsp.h

```diff
@@ -113,6 +115,8 @@ class LSPLoop {
     std::chrono::time_point<std::chrono::steady_clock> lastMetricUpdateTime;
     /** ID of the main thread, which actually processes LSP requests and performs typechecking. */
     std::thread::id mainThreadId;
+    /* If true, then the server is running across multiple threads. */
+    bool multithreadedMode = false;
```

Comment:
can this be `const`?

Reply:
Yeah, I can make it `const`. That'll also let me `ENFORCE` that we don't take the single threaded path in multithreaded mode (and vice versa).